main/src/ca/uwaterloo/flix/language/phase/typer/ConstraintSolverInterface.scala

```diff
@@ -37,24 +37,16 @@ object ConstraintSolverInterface {
     * Resolves constraints in the given definition using the given inference result.
     */
   def visitDef(defn: KindedAst.Def, infResult: InfResult, renv0: RigidityEnv, tconstrs0: List[TraitConstraint], econstrs0: List[EqualityConstraint], tenv0: TraitEnv, eqEnv0: EqualityEnv, root: KindedAst.Root)(implicit flix: Flix): (SubstitutionTree, List[TypeError]) = defn match {
-    case KindedAst.Def(sym, spec, _, _) =>
-      if (flix.options.xprinttyper.contains(sym.toString)) {
```

Comment:
This is too specialized. The options are:

- Use --xprint-phases and dump all the constraints (why not?) 
- Use a custom FlixListener
- Just readd the debugging as needed